docs/services-developer-guide.md

```diff
@@ -47,7 +47,7 @@ illustrates this. First, the configuration resources:
 1. A portable [resource class]. Portable resource classes are a pointer from a
    resource claim to a non-portable resource class that should be used for
    dynamic provisioning. [`RedisClusterClass`] is an example of a portable
-   resource claim.
+   resource class.
```

Comment:
D: I have been working on this project for almost a year and I still mix up class and claim ~80% of the time. Thanks for catching this!